.config/smtp.config.php

```diff
@@ -8,8 +8,15 @@
     'mail_smtpauth' => getenv('SMTP_NAME') && getenv('SMTP_PASSWORD'),
     'mail_smtpauthtype' => getenv('SMTP_AUTHTYPE') ?: 'LOGIN',
     'mail_smtpname' => getenv('SMTP_NAME') ?: '',
-    'mail_smtppassword' => getenv('SMTP_PASSWORD') ?: '',
     'mail_from_address' => getenv('MAIL_FROM_ADDRESS'),
     'mail_domain' => getenv('MAIL_DOMAIN'),
   );
+
+  if (getenv('SMTP_PASSWORD_FILE') && file_exists(getenv('SMTP_PASSWORD_FILE'))) {
+      $CONFIG['mail_smtppassword'] = rtrim(file_get_contents(getenv('SMTP_PASSWORD_FILE')));
```

Comment:
```suggestion
      $CONFIG['mail_smtppassword'] = trim(file_get_contents(getenv('SMTP_PASSWORD_FILE')));
```

Reply:
Fixed in [cc50d48](https://github.com/nextcloud/docker/pull/1471/commits/cc50d4801377cd6e9fa72fb0d14ee71a73bb7073).